### PR TITLE
Tweak revapi configuration

### PR DESCRIPTION
**Pull request by MarianMacik**
Hi

I have tweaked revapi configuration a little bit, together with versions update in droolsjbpm/droolsjbpm-build-bootstrap#303.

Thanks,

Marian

### CODE REVIEW

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by psiroky** [comment]
jenkins retest this

**Comment by psiroky** [comment]
jenkins retest this

File: kie-api/src/build/revapi-config.json [hunk 1]
```diff
@@ -1,5 +1,16 @@
 {
   "revapi" : {
+    "java" : {
+      "filter" : {
+        "classes" : {
+          "regex" : true,
+          /* Not used. The class is not referenced in any of the downstream (Drools, jBPM) code. The Activator itself was disabled in 2013.
+             Exclusion is here in order not to check the transitive classes from dependencies which this class used. This exclude can be removed once
+             this class is not present in the "oldVersion" of artifact, i.e. 7.0.0.Final */
```
Comment (psiroky) [inline kie-api/src/build/revapi-config.json]: I am afraid JSON does not support comment like these. Revapi seems to cope with that, but I guess we should fix this to make the file compliant to the JSON standard.

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
Unfortunately, JSON does not support multiline comments, but I have managed to put the comment in multiple key-value pairs. It should be OK now.

**Comment by MarianMacik** [comment]
Jenkins build has passed. This can be merged now.

**Comment by psiroky** [comment]
Merged as https://github.com/droolsjbpm/droolsjbpm-knowledge/commit/0337ec58db9804aafa87024d23271580cd932b98. I slightly changed the way the comments are structured, hope that's alright.

I've also noticed that Revapi forks the executions, which is bad. I am not sure if it was like that also before, or if the recent upgrade caused that. Nevertheless we need to fix it. Created https://issues.jboss.org/browse/DROOLS-1363 to track it.

**Pull request closed without merging**